docker-images/linux-nodejs-x64/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM 123124136734.dkr.ecr.us-east-1.amazonaws.com/aws-common-runtime/ubuntu-16.04:x64
+
+ENV NODE_VERSION="10.1.0"
+
+###############################################################################
+# NodeJS
+###############################################################################
+RUN set -ex \
```

Comment:
Should we use nvm and install 11 and 12 and test against those too? Maybe as a follow-up?

Reply:
That's a great idea... for later! Let's get this rolled out to the other CRTs first.